quickstarts/dockerdesktop.md

```diff
@@ -0,0 +1,105 @@
+Docker Desktop comes includes a standalone Kubernetes server and client, designed for local testing of Kubernetes.
+The Kubernetes cluster is started as a single-node cluster within a Docker container on your local system.
+
+# Installing the dependencies
+
+This quickstart assumes that you have the latest version of Docker Desktop, which you can get [here](https://docs.docker.com/desktop/).
+
+If you are running on Linux you'll need to install the `kubectl` binary separately, which you can get by following the instructions [here](https://kubernetes.io/docs/tasks/tools/).
+
+Once you have all the binary installed, make sure everything works:
+
+```shell
+# Validate kubectl if on Linux
+kubectl version
+```
+
+# Starting Kubernetes cluster
+
+This will start a local development cluster of Kubernetes with [Docker Desktop](https://docs.docker.com/desktop/kubernetes/) which runs in a container on your local machine.
+
+1. From the Docker Dashboard, select the Setting icon, or Preferences icon if you use a macOS.
+1. Select Kubernetes from the left sidebar.
+1. Next to Enable Kubernetes, select the checkbox.
+1. Select Apply & Restart to save the settings and then click Install to confirm.
+   This instantiates images required to run the Kubernetes server as containers, and installs the `/usr/local/bin/kubectl` command on your machine.
+
+
+# Applying Strimzi installation file
+
+Before deploying Strimzi Kafka operator, let's first create our `kafka` namespace:
+
+```shell
+kubectl create namespace kafka
+```
+
+Next we apply the Strimzi install files, including `ClusterRoles`, `ClusterRoleBindings` and some **Custom Resource Definitions** (`CRDs`). The CRDs define the schemas used for declarative management of the Kafka cluster, Kafka topics and users.
+
+```shell
+kubectl create -f 'https://strimzi.io/install/latest?namespace=kafka' -n kafka
+```
+
+This will be familiar if you've installed Strimzi before.
+Note how we set all the `namespace` references in downloaded .yaml file to `kafka`. By default they are set to `myproject`.
+But we want them all to be `kafka` because we decided to install the operator into the `kafka` namespace, which we achieve by specifying `-n kafka` when running `kubectl create` ensuring that all the definitions and the configurations are installed in `kafka` namespace rather than the `default` namespace.
```

Comment:
I think this is rather confusing.

Reply:
This is actually copied directly from the Kubernetes Kind quickstart. I think it is good to explain what's happening with the namespace, which part do you think is confusing? Or do you think we shouldn't be explaining it at all?

Reply:
* `Note how we set all the namespace references in downloaded .yaml file to kafka.` => what exactly should be noted? How the serverless function on the website replaces `myproject` for `kafka` in the YAML they actually don' see because it is passed directly to `kubectl`? Or is that about the `-n kafka` which however only does the small part of the task? I'm not really sure what should be noted here.
* The whole way how it mixes 3 different namespaces `kafka`, `myproject` and `default` namespaces is IMHO confusing as well. So TBH, I'm a bit sceptical that this will mean anything to someone who is completely new.

Reply:
I've tweaked Paul's suggestion and used that. See what you think